.github/workflows/release-scheduler.yml

```diff
@@ -34,8 +34,10 @@ jobs:
           BODY: |
             This is an automated reminder that a new release for eBPF is scheduled to be created:
 
-            1. Follow instructions in [ReleaseProcess.md](https://github.com/microsoft/ebpf-for-windows/blob/main/docs/ReleaseProcess.md).
-            1. Sync the internal `mscodehub` and follow the internal release instructions in `\.internal\docs\ReleaseEbpfRedistProcess.md`.
+            - [ ] Ensure SPGO data is regenerated and added to the repo using performance test suite.
```

Comment:
For clarity, how to know that the SPGO data was not regenerated? If it doesn't, what are the steps to force regenerate?

Reply:
```cmd
git log -1 EbpfCore.spd
```
Then look at the time stamp of the last commit.

Reply:
I think Anurag is suggesting that the text "Ensure..." isn't clear enough for someone to follow and how to do so should be added to the actual text in the PR.